src/CoreBluetooth/CBUUID.cs

```diff
@@ -84,7 +84,7 @@ public override bool Equals (object obj)
 		// this fixes a compiler warning: CS0661: `CoreBluetooth.CBUUID' defines operator == or operator != but does not override Object.GetHashCode()
 		public override int GetHashCode ()
 		{
-			return base.GetHashCode ();
+			return HashCode.Combine (base);
```

Comment:
Ah, in this case you want to call base.GetHashCode, since according to the comment above the method the base implementation is good enough, and we only have the code here to silence a compiler warning.